ui/app/templates/vault/cluster/access/identity/show.hbs

```diff
@@ -34,7 +34,7 @@
 </div>
 <Toolbar>
   <ToolbarActions>
-    {{! template-lint-configure simple-unless "warn" }}
+    {{! template-lint-disable simple-unless }}
     {{#unless (or (and (eq this.model.identityType "group") (eq this.model.type "internal")) this.model.alias)}}
```

Comment:
blah, that's a lot of logic to work through.

Reply:
Haha ya I had to just disable the rule and not risk a regression here. :exploding_head: